README.rst

```diff
@@ -27,38 +26,111 @@ Installation on Open edX Devstack
   pip install -e /edx/src/openedx-lti-tool-plugin
   /edx/app/edxapp/edx-platform/manage.py lms migrate openedx_lti_tool_plugin # Run plugin migrations
 
-- Set the lms setting OLTITP_ENABLE_LTI_TOOL to True
+4. Set the lms setting OLTITP_ENABLE_LTI_TOOL to True.
+
+.. code-block:: bash
+  echo 'OLTITP_ENABLE_LTI_TOOL=True' >> ~openedx/edx-platform/lms/envs/devstack_docker.py
+
+5. Restart the LMS.
 
 Development Setup
 =================
 
-- Clone this repository
+1. Clone this repository.
 
 .. code-block:: bash
 
   git clone git@github.com:Pearson-Advance/openedx-lti-tool-plugin.git
 
-- Set up a virtualenv
+2. Set up a virtualenv.
 
 .. code-block:: bash
 
   cd openedx-lti-tool-plugin
   virtualenv venv
   source venv/bin/activate
 
-- Install development dependencies.
+3. Install development dependencies.
 
 .. code-block:: bash
 
   make dev-requirements
 
-Settings
-========
+LTI 1.3 Resource Launch Setup
+=============================
+
+1. (Optional) Expose the LMS service to an external domain.
```

Comment:
This step might not be clear for all people, could you please elaborate a little bit more?, don't know if you can name some of the tools you have used. @kuipumu